### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by michaljurecko**
Part of: https://keboola.atlassian.net/browse/BA-56

Changes:
- Separated files by the service type.
- Helm values are not processed by envsubstr.

Before:
image

After:
image

### CODE REVIEW

File: provisioning/templates-api/kubernetes/build.sh [hunk 1]
```diff
@@ -4,13 +4,15 @@ set -Eeuo pipefail
 # CD to the script directory
 cd "$(dirname "$0")"
 
+# Namespace
+envsubst < templates/namespace.yaml > deploy/namespace.yaml
+
```
Comment (michaljurecko) [inline provisioning/templates-api/kubernetes/build.sh:9]: Namespace je prva vec - presunute vyssie, co treba nastavit (aj ked tu sa iba nahradzuje sablona).

File: provisioning/templates-api/common.sh [hunk 2]
```diff
@@ -39,4 +39,5 @@ helm upgrade \
 
 # API
 kubectl apply -f ./kubernetes/deploy/api/config-map.yaml
+kubectl apply -f ./kubernetes/deploy/api/pdb.yaml
 kubectl apply -f ./kubernetes/deploy/api/deployment.yaml
```
Comment (michaljurecko) [inline provisioning/templates-api/common.sh:44]: Aby mal kazdy kind svoj samostatny subor.

File: provisioning/templates-api/common.sh [hunk 3]
```diff
@@ -35,6 +35,7 @@ helm upgrade \
   --version "$ETCD_HELM_CHART_VERSION" \
   --values ./kubernetes/deploy/etcd/values.yaml \
   --namespace templates-api \
+  --set "replicaCount=$TEMPLATES_API_ETCD_REPLICAS" \
```
Comment (michaljurecko) [inline provisioning/templates-api/common.sh:38]: Namiesto nahradzovania ENV v `values.yaml`, nastavime tuto hodnotu priamo cez HELM.
Vramci cron job-u pre defragmentaciu mi to robilo problemy, ked som potreboval pouzit znak `$`.

**Review by michaljurecko — COMMENTED** [review]
Okomentoval som to po commitoch.

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**